docs/en/operations/settings/settings.md

```diff
@@ -1262,4 +1262,34 @@ Possible values:
 
 Default value: 16.
 
+## background_move_pool_size {#background_move_pool_size}
+
+Sets the number of threads performing background moves in table engines. This setting is applied at ClickHouse server start and can’t be changed in a user session.
+
+Possible values:
+
+-   Any positive integer.
+
+Default value: 8.
+
+## background_schedule_pool_size {#background_schedule_pool_size}
+
+Sets the number of threads performing background tasks for replicated tables, Kafka streaming, DNS cache updates. This setting is applied at ClickHouse server start and can’t be changed in a user session.
```

Comment:
I think we should links to replication, Kafka table engine and setting that regulates DSN cache updates https://clickhouse.tech/docs/en/operations/server-configuration-parameters/settings/#server-settings-dns-cache-update-period